tests/e2e/stablehlo_ops/constant.mlir

```diff
@@ -0,0 +1,26 @@
+func.func @high_rank () {
+  %dense = stablehlo.constant dense<[[[1, 2, 3], [4, 5, 6]], [[7, 8, 9], [10, 11, 12]]]> : tensor<2x2x3xi32>
+  check.expect_eq_const(%dense, dense<[[[1, 2, 3], [4, 5, 6]], [[7, 8, 9], [10, 11, 12]]]> : tensor<2x2x3xi32>) : tensor<2x2x3xi32>
+
+  // %splat = stablehlo.constant dense<1> : tensor<2x2x3xi32>
+  // check.expect_eq_const(%splat, dense<1> : tensor<2x2x3xi32>) : tensor<2x2x3xi32>
+  return
+}
```

Comment:
This file has some test cases commented out. Can they be uncommented?

Reply:
This is copied from the XLA version of these test -- I'd rather keep it as is.